packages/flutter/lib/src/material/button.dart

```diff
@@ -10,8 +10,28 @@ import 'ink_well.dart';
 import 'material.dart';
 import 'theme.dart';
 
-enum ButtonColor { normal, accent }
+/// Whether a button should use the accent color for its text.
+///
+/// See also:
+///
+///  * [ButtonTheme]
+///  * [RaisedButton]
+///  * [FlatButton]
+enum ButtonColor {
+  /// The button should use the normal color (e.g., black or white depending on the [ThemeData.brightness]) for its text.
```

Comment:
I assume that what you've specified matches the implementation but it seems like the button's color should affect its Icon children as well.